internal/handler/wallet/getall.go

```diff
@@ -41,12 +41,35 @@ func (h *walletGetAll) Handle(params operations.AccountListParams) middleware.Re
 			})
 	}
 
-	for i := 0; i < len(wallets); i++ {
-		modelWallet := createModelWallet(wallets[i])
+	for i := 0; i < len(walletsWithoutError); i++ {
+		status := "ok"
```

Comment:
we should use a constant variable, the same as the one we use when creating a new account